speaking-portal-project/src/test/file_check.ts

```diff
@@ -0,0 +1,104 @@
+import { chdir, cwd } from 'node:process'
+import fs from 'fs'
+
+export async function checkFiles(audio_file_name : string, text_file_name : string){
+    /**
+     * Checks the file in the Rhubarb directory: existence, read permissions, valid files .wav and .txt
+     *
+     * @remarks
+     * Make sure to call this method in a try-catch statement, as all its helper methods throw errors or return true
+     *
+     * @param file_name - The name of the file to check
+     */
+    chdir('./rhubarb') // should we do this here?
+
+    // Check Audio File
+    checkFileExistence(audio_file_name)
+    checkFileReadAccess(audio_file_name)
+    checkWavFile(audio_file_name)
+
+    // Check Text File
+    checkFileExistence(text_file_name)
+    checkFileReadAccess(text_file_name)
+    checkTextFile(text_file_name)
+
+    // TODO: chdir() back to original directory? or do we do this after running Rhubarb? should it be here, in index.ts or in rhubarb.ts?
```

Comment:
Each time we change directory to the rhubarb directory, we should be switching back to the main directory